addons/l10n_it_edi/models/account_edi_format.py

```diff
@@ -1012,11 +1012,15 @@ def _l10n_it_post_invoices_step_2(self, invoices):
         return to_return
 
     def _post_fattura_pa(self, invoice):
-        # OVERRIDE
-        if not invoice.l10n_it_edi_transaction:
-            return self._l10n_it_post_invoices_step_1(invoice)
-        else:
-            return self._l10n_it_post_invoices_step_2(invoice)
+        # TODO: since batching is now possible, we should change
+        # the name of the invoice parameter to invoices
+        invoices1 = invoice.filtered(lambda x: not x.l10n_it_edi_transaction)
+        result1 = self._l10n_it_post_invoices_step_1(invoices1)
+
+        invoices2 = invoice.filtered(lambda x: x.l10n_it_edi_transaction)
```

Comment:
What if they got an l10n_it_edi_transaction in the meantime and what happens when in the return result then?

Reply:
We discussed about it -  the batching avoids mixing the two groups, and we're changing the implementation to something mutually exclusive.